File: marketplace-frontend/src/app/main/content/_model/stored-chart.ts
export class StoredChart {
  id: string;
  userId: string;
  title: string;
  type: string;
  data: string;

  constructor(title: string, type: string, data: string, userId: string) {
    this.title = title;
    this.type = type;
    this.data = data;
    this.userId = userId;
  }
}

File: marketplace/marketplace-model/src/main/java/at/jku/cis/iVolunteer/model/diagram/StoredChart.java
package at.jku.cis.iVolunteer.model.diagram;

import org.springframework.data.mongodb.core.mapping.Document;

@Document
public class StoredChart {
	private String id;
	private String userId;
	private String title;
	private String type;
	private String data;

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getUserId() {
		return id;
	}

	public void setUserId(String userId) {
		this.userId = userId;
	}

	public String getTitle() {
		return title;
	}

	public void setTitle(String title) {
		this.title = title;
	}

	public String getType() {
		return type;
	}

	public void setType(String type) {
		this.type = type;
	}

	public String getData() {
		return data;
	}

	public void setData(String data) {
		this.data = data;
	}
}
